reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -531,6 +531,7 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
         val pathExcludes = getPathExcludes(id, scanResult.provenance)
         val licenseFindingCurations = getLicenseFindingCurations(id, scanResult.provenance)
         val curatedFindings = curationsMatcher.applyAll(scanResult.summary.licenseFindings, licenseFindingCurations)
+            .mapNotNull { it.curatedFinding }.toSet()
```

Comment:
Nit: Could be `.mapNotNullTo(mutableSetOf()) { it.curatedFinding }` to avoid the creation of a temporary list.

Reply:
Done.